index.html

```diff
@@ -2812,6 +2812,15 @@ <h2>Algorithms</h2>
 [=map=] with a `proof` [=map/key=].
       </p>
 
+      <p>
+In addition to the algorithms below, implementers MAY implement additional
+reasonable defaults and safeguards to mitigate things such as developer error,
+excessive resource consumption, or attack models that they would like to protect
```

Comment:
Hmm, this reads (to me) as "mitigate ... attack models", which doesn't seem right, so slight change:

```suggestion
excessive resource consumption, or other attacks that they would like to protect
```

Reply:
"other attacks" would mean that "developer error" and "excessive resource consumption" are attacks, which I think is incorrect. See my larger suggestion for this paragram...